.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "15:15"
+    timezone: "Europe/Sofia"
+  open-pull-requests-limit: 10
+- package-ecosystem: npm
+  directory: "packages/cryptography"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  directory: "packages/proto"
```

Comment:
This seems to be missing a `- package-ecosystem: npm`

Reply:
Good point, added :+1: